index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Image Slider animation</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="slider">
+        <div class="slide slide1">
+          <div class="caption">
+            <h2>Who Am I?</h2>
+            <p>
+              Hey, This is Nehal Shaikh. I Welcome you on my website. Sit back
+              and relax.
+            </p>
+          </div>
+        </div>
+        <div class="slide slide2">
+          <div class="caption">
+            <h2>Qualification Background</h2>
+            <p>
+              I am located in Mumbai and have completed my graduation in
+              Bachelors of Mass Media from Mumbai university.
+            </p>
+          </div>
+        </div>
+        <div class="slide slide3">
+          <div class="caption">

```

Comment:
You can also used your images here with your class name.

Reply:
Definitely but i found the approach from Css file would be fun.